docs/general/ad-filtering/create-own-filters.md

```diff
@@ -332,13 +332,12 @@ AdGuard will try to close the browser tab with any address that matches a blocki
 
 > **Compatibility with different versions of AdGuard**
 >
-> `$popup` modifier is reliable in AdGuard Browser Extension.
-> In AdGuard for Safari and iOS, `$popup` rules will simply block the page right away.
->
-> When it is used with AdGuard for Windows, Mac and Android, it is very unreliable.
-> `$popup` modifier applies the `document` content type with a special flag which is passed to a blocking page.
-> Blocking page itself can do some checks and close the window if it is really a popup.
-> Otherwise, page should be loaded.
+> 1. `$popup` modifier works best in AdGuard Browser Extension.
```

Comment:
I guess 1. is not supposed to be here and in two strings below)

Reply:
it looks better 
https://uploads.adguard.org/slbdzdrvtmenn.png

comparing to plain text
https://uploads.adguard.org/slbdzdrvtajrb.png

different behaviour for different list items

Reply:
but it's `1.` at each point

Reply:
but it's `1` at each point

Reply:
but it's `1` at each point

Reply:
it is fine
https://google.github.io/styleguide/docguide/style.html#lists